apostrophecms/apostrophe
# display placeholders for custom widgets

Depends on #3882

## Summary

Set the placeholder values for any field that has some, right before the widgets rendering.

## What are the specific steps to test this change?

On testbed:

- add `topic` widget to some area
- add the `initialModal: false` option to `topic-widget`
- add a placeholder to the `title` field.
👉 you should have the placeholder value when you add topic widgets 
- edit the widget title via the edit modal
👉 now you should have the title you gave, **not the placeholder**

👉 if a field has a `def` value, it should take over the placeholder.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
